src/java.base/share/classes/java/security/GuardedObject.java

```diff
@@ -52,10 +54,17 @@
     @java.io.Serial
     private static final long serialVersionUID = -5240450096227834308L;
 
+    /**
+     * The object we are guarding
+     */
     @SuppressWarnings("serial") // Not statically typed as Serializable
-    private Object object; // the object we are guarding
+    private Object object;
+
+    /**
+     * The guard object
```

Comment:
add a period?

Reply:
Probably worth doing as it's the first sentence.